maintenance/wikia/VideoHandlers/nfcNormalizeVideos.php

```diff
@@ -96,8 +105,25 @@ protected static function updateVideo( $video, DatabaseMysql $db, $test, $verbos
 			. "for: $originalName img_sha1: {$video->img_sha1}\n";
 
 		if ( $verbose ) {
-			echo "\tNew title: $name\n";
+			echo "\tNew title: $name"
+				. "\n\tNew metadata: $metadata\n";
 		}
 	}
 
+	/**
+	 * Get Normalized metadata in PHP-serialized form
+	 *
+	 * @param stdClass $video
+	 * @return string
+	 */
+	protected static function getNormalizedMetadata( $video ) {
```

Comment:
Any reason to do this field by field rather than via the `json_decode( json_encode )` method used in `VideoFileUploader.class.php`?